src/content/en/updates/2017/06/chrome-60-deprecations.md

```diff
@@ -91,6 +91,15 @@ The new behavior is implemented in Chrome 61.
 
 ## JavaScript
 
+### Deprecate and remove RTCPeerConnection.getStreamById()
+
+Nearly two years ago, `getStreamById()` was [removed from the Web RTC spec](https://github.com/w3c/webrtc-pc/pull/18). Most other browsers have removed already removed this from their implementations. Though this function is believed to be little-used, it's also believed there is some minor interoperability risk with Edge and WebKit-based browsers *other than* Safari where `getSTreamById()` is still supported. Developers needing an alternative implementation can find example code in the Intent to Remove, below.
```

Comment:
@jpmedley WebRTC is usually written as one word. (It's great how inconsistent WebFoo spacing is...)